web-server/v0.4/src/pages/Dashboard/Summary.js

```diff
@@ -713,179 +394,175 @@ class Summary extends ReactJS.Component {
-                  <p className="list-group-item-text">{host.tools['proc-interrupts']}</p>
-                </li>
-                <li className="list-group-item">
-                  <h5 className="list-group-item-heading">proc-vmstat</h5>
-                  <p className="list-group-item-text">{host.tools['proc-vmstat']}</p>
-                </li>
-                <li className="list-group-item">
-                  <h5 className="list-group-item-heading">sar</h5>
-                  <p className="list-group-item-text">{host.tools.sar}</p>
-                </li>
-                <li className="list-group-item">
-                  <h5 className="list-group-item-heading">pidstat</h5>
-                  <p className="list-group-item-text">{host.tools.pidstat}</p>
-                </li>
-                <li className="list-group-item">
-                  <h5 className="list-group-item-heading">turbostat</h5>
-                  <p className="list-group-item-text">{host.tools.turbostat}</p>
-                </li>
-                <li className="list-group-item">
-                  <h5 className="list-group-item-heading">iostat</h5>
-                  <p className="list-group-item-text">{host.tools.iostat}</p>
-                </li>
-              </div>
-            ))}
-          </Card>
-        ),
-        toc: (
-          <Card title="Table of Contents" style={{ marginTop: 32 }}>
-            <Table columns={columns} dataSource={tocTree} defaultExpandAllRows />
-          </Card>
-        ),
-      };
-
-      return (
-        <div style={{ display: 'flex', flexDirection: 'column' }}>
-          <div>
-            <PageHeaderLayout
-              title={selectedResults['run.name']}
-              content={
-                <Tag color="blue" key={selectedController}>
-                  {'controller: ' + selectedController}
-                </Tag>
-              }
-              tabList={tabList}
-              tabActiveKey={activeTab}
-              onTabChange={this.onTabChange}
-            />
-            {contentList[activeTab]}
-          </div>
+              <li className="list-group-item">
+                <h5 className="list-group-item-heading">turbostat</h5>
+                <p className="list-group-item-text">{host.tools.turbostat}</p>
+              </li>
+              <li className="list-group-item">
+                <h5 className="list-group-item-heading">iostat</h5>
+                <p className="list-group-item-text">{host.tools.iostat}</p>
+              </li>
+            </div>
+          ))}
+        </Card>
+      ),
+      toc: (
+        <Card title="Table of Contents" style={{ marginTop: 32 }}>
+          <Table columns={columns} dataSource={tocTree} defaultExpandAllRows />
+        </Card>
+      ),
+    };
+
+    return (
+      <div style={{ display: 'flex', flexDirection: 'column' }}>
+        <div>
+          <PageHeaderLayout
+            title={selectedResults['run.name']}
```

Comment:
The name of the run is not being rendered as we are not referring to the first run item in the `selectedResults` array.

Reply:
@gurbirkalsi, fixed.